hardhat.config.ts

```diff
@@ -43,9 +44,9 @@ module.exports = {
   networks: {
     generic: {
       // set URL for external network
-      url: `${ process.env.URL }`,
+      url: `${process.env.URL}`,
```

Comment:
Going to need padding inside of these brackets.  Let's set a prettier or eslint rule at some point.

Reply:
Prettier doesnt have an option to add padding inside template litterals, only for brackets of objects